web/src/components/apps/Dashboard.tsx

```diff
@@ -785,7 +886,7 @@ class Dashboard extends Component {
             {this.state.viewAirgapUpdateError && (
               <Modal
                 isOpen={this.state.viewAirgapUpdateError}
-                onRequestClose={this.toggleViewAirgapUpdateError}
+                onRequestClose={() => this.toggleViewAirgapUpdateError()}
```

Comment:
`toggleViewAirgapUpdateError` sometimes takes a string but never takes a mouseEvent. wrapping this function so it's clear that we're not trying to use what `onRequestClose` calls with